### ISSUE TITLE
Hardware related metrics

### ISSUE THREAD

**Issue by anvitpatil** [microsoft/nni#4334]
It is possible to use hardware-related metrics such has flops to specialize the search in NNI for a target hardware? I want to use a hardware related metric along with accuracy to tune my model.

**Comment by JiahangXu** [comment]
Thanks for raising the question. NNI support latency and flops as hardware-related metrics by now. 

As for latency, you can use the package `nn-Meter` to predict the latency of the target model with four provided devices. Guidance could be seen at https://github.com/microsoft/nn-Meter. NNI also provide examples to utilize `nn-Meter` for hardware-aware NAS (Document: https://nni.readthedocs.io/en/stable/NAS/HardwareAwareNAS.html).

As for FLOPS, NNI provides an API as `count_flops_params()` from `nni.compression.pytorch.utils.counter` to count FLOPs and params of the given model.

Hope these information could help.

**Comment by scarlett2018** [comment]
closing the issue as the answer has been provided, feel free to reopen the issue if there are any further questions.
